en/editor/extension/messages.md

```diff
@@ -14,9 +14,9 @@ Message interaction is divided into two situations:
 
 The first is called ordinary messages, and the second is called broadcast messages.
 
-### General news
+### General News
 
-It can be understood as a kind of external api, for example, "scene editor" defines a "message" API-query-node (query-node)
+It can be understood as a kind of external api, for example, **scene editor** defines a **message** API-query-node (query-node).
 
 ```json5
```

Comment:
\`\`\`json

Reply:
https://github.com/cocos-creator/docs-3d/pull/377

using json5 here...

Reply:
panda decided in WeChat